examples/visual_voice_activity_detection/ava_dataset.py

```diff
@@ -0,0 +1,189 @@
+import os
+import urllib.request
+import cv2
+import numpy as np
+import tensorflow as tf
+import logging
+
+class AvaDataset:
+    def __init__(self,
+                 root_dir="ava_data",
+                 file_list_url="https://s3.amazonaws.com/ava-dataset/annotations/ava_speech_file_names_v1.txt",
+                 video_url_template="https://s3.amazonaws.com/ava-dataset/trainval/{}",target_fps=25,resize=None, log_level=logging.INFO):
+        self.root_dir = root_dir
+        self.video_dir = os.path.join(root_dir, "videos")
+        self.csv_dir = os.path.join(root_dir, "annotations")
```

Comment:
I ran (still running) the download of all videos but I think there is no code to download the CSV files with the annotations.